pyo3-derive-backend/src/pymethod.rs

```diff
@@ -246,6 +249,21 @@ pub fn impl_wrap_static(cls: &syn::Type, spec: &FnSpec<'_>) -> TokenStream {
     }
 }
 
+/// Generate a wrapper for initialization of a class attribute.
+/// To be called in `pyo3::pyclass::initialize_type_object`.
+pub fn impl_wrap_class_attribute(cls: &syn::Type, spec: &FnSpec<'_>) -> TokenStream {
+    let name = &spec.name;
+    let cb = quote! { #cls::#name() };
+    let into_result = quote! { pyo3::derive_utils::IntoPyResult::into_py_result };
+
+    quote! {
+        fn __wrap(py: pyo3::Python<'_>) -> pyo3::PyResult<*mut pyo3::ffi::PyObject> {
```

Comment:
Since this `__wrap` is not an FFI function(=it is called only in PyO3, not CPython itself), how about using pyo3's objects?
I.e., 
```rust
fn __wrap(py: pyo3::Python<'_>) -> pyo3::PyResult<PyObject> { ... }
```
or so.
Note that you need to use `into_ptr` instead of `as_ptr` when you set the `PyObject` to the tp_dict.